sdk/storage/storage-file-datalake/src/models.ts

```diff
@@ -619,9 +631,76 @@ export interface FileFlushOptions extends CommonOptions {
   pathHttpHeaders?: PathHttpHeaders;
 }
 
-export interface FileCreateOptions extends PathCreateOptions {}
+export interface FileCreateOptions extends PathCreateOptions { }
+
+export interface FileCreateResponse extends PathCreateResponse { }
 
-export interface FileCreateResponse extends PathCreateResponse {}
+export interface FileParallelUploadOptions extends CommonOptions {
+  // For all.
+  abortSignal?: AbortSignalLike;
+  conditions?: DataLakeRequestConditions;
+
+  // For create and flush.
+  pathHttpHeaders?: PathCreateHttpHeaders;
+
+  // For create.
+  metadata?: Metadata;
+  permissions?: string; // TODO: model or string?
+  umask?: string; // TODO: model or string?
+
+  // For append.
+  onProgress?: (progress: TransferProgressEvent) => void;
+
+  // For flush.
+  close?: boolean;
+
+  // For parallel transfer control.
+
+  /**
+   * The size of the first range request in bytes. Data smaller than this limit will
+   * be transferred in a single request. Data larger than this limit will be transferred
+   * in chunks of size chunkSize. Its default and max value is FILE_MAX_INITIAL_TRANSFER_SIZE.
+   *
+   * @type {number}
+   * @memberof FileParallelUploadOptions
+   */
+  initialTransferSize?: number;
+
+  /**
+   * The size of data that will be transferred in parallel. Its default value is FILE_UPLOAD_DEFAULT_CHUNK_SIZE.
+   * Its max value is FILE_UPLOAD_MAX_CHUNK_SIZE.
+   *
+   * @type {number}
+   * @memberof FileParallelUploadOptions
+   */
+  chunkSize?: number;
+  /**
+   * Max concurrency of parallel uploading. Must be >= 0. Its default value is DEFAULT_HIGH_LEVEL_CONCURRENCY.
+   * 
+   * @type {number}
+   * @memberof FileParallelUploadOptions
+   */
+  maxConcurrency?: number;
+}
+
+export interface FileReadToBufferOptions extends CommonOptions {
+  abortSignal?: AbortSignalLike;
+  conditions?: DataLakeRequestConditions;
+
+  onProgress?: (progress: TransferProgressEvent) => void;
+
+  /**
+   * How many retries will perform for each read when the original chunk read stream ends unexpectedly.
+   * Above kind of ends will not trigger retry policy defined in a pipeline,
+   * because they doesn't emit network errors.
+   *
+   * @type {number}
+   * @memberof FileReadToBufferOptions
+   */
+  maxRetryRequestsPerChunk?: number;
```

Comment:
Where is the method implementation in PR?

Reply:
Most of them are in sdk/storage/storage-file-datalake/src/clients.ts.